src/testing/jest/jest-27-and-under/matchers/screenshot.ts

```diff
@@ -12,7 +12,15 @@ export function toMatchScreenshot(compare: d.ScreenshotDiff, opts: d.MatchScreen
   }
 
   if (typeof compare.mismatchedPixels !== 'number') {
-    throw new Error(`expect toMatchScreenshot() value is not a screenshot compare`);
+    throw new Error(
+      `expect toMatchScreenshot() value is not a valid screenshot compare object - 'mismatchedPixels' has type '${typeof compare.mismatchedPixels}', but should be a number`,
```

Comment:
This error message before wasn't the greatest - since I was already doing the same thing for `deviceScaleFactor` I decided to copy/paste/replace that error msg here too